knowagedao/src/main/java/it/eng/spagobi/tools/dataset/dao/DataSetDAOImpl.java

```diff
@@ -1539,6 +1540,7 @@ public List<IDataSet> loadFilteredDatasetList(Integer offset, Integer fetchSize,
 			transaction = session.beginTransaction();
 
 			Query listQuery = session.createQuery(sb.toString());
+			listQuery.setParameter(":entityName", "ds.");
```

Comment:
Entity aliases cannot be injected as parameters

Reply:
Hello @davide-zerbetto, we are injecting just the "ds." string, this is more like a prefix, I think it is needed because the tool returned this file like it still has an issue, but I can't find what else could be the problem. Do you think the same?

Reply:
"ds" is the alias for SbiDataSet hibernate entity, see line 1504; it does not make sense both to have it as a separate variable and to replace it as a named parameter. Remove the variable completely and use the ds alias wherever it is needed in the query strings